### PR TITLE
Mobile Center Sample

### PR DESCRIPTION
**Pull request by MatkovIvan**

### CODE REVIEW

**Review by jaeklim — APPROVED** [review]
Looks good to me.

File: Mobile Center Sample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+					"DEBUG=1",
+					"$(inherited)",
+				);
+				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
+				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				MACOSX_DEPLOYMENT_TARGET = 10.12;
```
Comment (jaeklim) [inline Mobile Center Sample.xcodeproj/project.pbxproj]: Minimum target is 10.9.

File: Mobile Center Sample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				GCC_C_LANGUAGE_STANDARD = gnu99;
+				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_WARN_64_TO_32_BIT_CONVERSION = YES;
+				GCC_WARN_ABOUT_RETURN_TYPE = YES_ERROR;
+				GCC_WARN_UNDECLARED_SELECTOR = YES;
+				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
+				GCC_WARN_UNUSED_FUNCTION = YES;
+				GCC_WARN_UNUSED_VARIABLE = YES;
+				MACOSX_DEPLOYMENT_TARGET = 10.12;
```
Comment (jaeklim) [inline Mobile Center Sample.xcodeproj/project.pbxproj]: Minimum target is 10.9.

**Review by jaeklim — CHANGES_REQUESTED** [review]
Minor changes.

File: Podfile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+target 'Mobile Center Sample' do
```
Comment (iamclement) [inline Podfile:1]: You probably want the specify the min Mac version supported, which is 10.9 for MC SDK

File: Mobile Center Sample.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+		F85F38691F5FED2C008E4157 /* Project object */ = {
+			isa = PBXProject;
+			attributes = {
+				LastSwiftUpdateCheck = 0830;
+				LastUpgradeCheck = 0830;
+				ORGANIZATIONNAME = Microsoft;
+				TargetAttributes = {
+					F85F38701F5FED2C008E4157 = {
+						CreatedOnToolsVersion = 8.3.3;
+						DevelopmentTeam = UBF8T346G9;
```
Comment (iamclement) [inline Mobile Center Sample.xcodeproj/project.pbxproj:132]: We probably want this to be empty, people will add their team themselves.

**Review by iamclement — CHANGES_REQUESTED** [review]
Some minor comments

**Comment by MatkovIvan** [comment]
See #2 without using `storyboard`s for compatibility with `10.9`

**Pull request closed without merging**